app/products/calls/connection/foreground_service.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import VIForegroundService from '@voximplant/react-native-foreground-service';
+
+import {logError} from '@utils/log';
+
+const channelConfig = {
+    id: 'calls_channel',
+    name: 'Mattermost',
+    description: 'Mattermost Calls microphone while app is in the background',
+    enableVibration: false,
+};
+
+export const foregroundServiceStart = async () => {
+    await VIForegroundService.getInstance().createNotificationChannel(channelConfig);
```

Comment:
Why create the notification channel every time it starts? Creating the channel once should be enough

Reply:
Calling it again with the same params is a noop, but just to be clean I separated it into a setup call. 👍